smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/CreatePageAsAuthorUserIT.java

```diff
@@ -75,10 +77,10 @@ public void testCreatePageAsAuthor() throws InterruptedException, ClientExceptio
         String pageName = "testpage_" +  UUID.randomUUID();
         String pagePathExpected = temporaryPage.getParentPath() + "/" + pageName;
         String pagePath = pagePathExpected;
-        try {
-            SlingHttpResponse response = userRule.getClient().createPageWithRetry(pageName, "Page created by CreatePageAsAuthorUserIT",
-                    temporaryPage.getParentPath(), "", MINUTES.toMillis(1), 500, HttpStatus.SC_OK, HttpStatus.SC_UNAUTHORIZED);
-            if (null != response && response.getStatusLine().getStatusCode() == HttpStatus.SC_UNAUTHORIZED) {
+        try (SlingHttpResponse response = userRule.getClient().createPageWithRetry(pageName, "Page created by CreatePageAsAuthorUserIT",
```

Comment:
can you split the `(` and `)` on different lines like above, to keep consistency?